### DISCUSSION TITLE
Any suggestions for the color scheme? Only the hue, not the layout.

### DISCUSSION THREAD

**Discussion by haonguyen97ftu** [alshedivat/al-folio#1639] [Q&A] [answered]
Do you have any suggestions for the color scheme of the theme? I'm not looking for any UI modifications, just the color palette.

**Comment by george-gca** [comment, marked as answer]
It is a good idea to use sites that [help choosing the best color combinations](https://www.google.com/search?q=choosing+color+palettes+for+site). [This one](https://www.upwork.com/resources/how-to-choose-color-scheme-for-website) has a few suggestions.
